docs/contributor/release-process.md

```diff
@@ -1,50 +1,50 @@
-# Backend - Release Workflow Documentation
+# Backend - Release Workflow
 
-This documentation provides a step-by-step guide to the release process workflow for building and pushing a Go project using GitHub Actions. This workflow is triggered on pushes to the `main` branch and tags, specifically for changes within the `assistant/backend` directory.
+This document provides a step-by-step guide to the release process workflow for building and pushing a Go project using GitHub Actions. This workflow is triggered on pushes to the `main` branch and tags, specifically for changes within the `assistant/backend` directory.
 
 ## Prerequisites
 
-- PR to the `main` branch:
-  - label: `lgtm`
-  - Approved by at least one reviewer
-  - Passes all checks (Bckend E2E tests, Linting, Git Leaks)
-
-## Files and folders
+A pull request (PR) to the `main` branch must:
+  * Have the `lgtm` label
+  * Get approval from at least one reviewer <!--code owner?-->
+  * Pass all checks (Bckend E2E tests, Linting, Git Leaks)
 
+## Files and Folders
+<!--what is this list?-->
 - GitHub Action Workflow: `.github/workflows`
 - Backend code: `assistant/backend`
 - Kubernetes related scripts and deployment files: `.scripts/kubernetes`
 - Shell scripts (k3s installation, deployment verification, test scripts): `.scripts/shell`
 
 ## Secrets
-
+<!--what is this list?-->
```

Comment:
Same thing as above, I am not sure. But as this uses `JFROG` I'm pretty sure this is also outdated and can be completely removed.